src/autoscaler/api/cred_helper/rpc.go

```diff
@@ -0,0 +1,64 @@
+package cred_helper
+
+import (
+	"autoscaler/models"
+	"net/rpc"
+)
+
+// Here is an implementation that talks over RPC
+type CredentialsRPC struct {
+	client *rpc.Client
+}
+
+func (g *CredentialsRPC) Create(appId string, userProvidedCredential *models.Credential) (*models.Credential, error) {
+	var resp string
+	err := g.client.Call("Plugin.Create", new(interface{}), &resp)
+	if err != nil {
+		// You usually want your interfaces to return errors. If they don't,
+		// there isn't much other choice here.
+		panic(err)
```

Comment:
hmm will this kill the plugin ?

i.e. the credentials plugin will stop working for that instance ?